app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -198,7 +205,7 @@ export default {
       }
       if (this.isATwitterInbox) {
         if (this.conversationType === 'tweet') {
-          return MESSAGE_MAX_LENGTH.TWEET;
+          return MESSAGE_MAX_LENGTH.TWEET - this.replyToUserLength - 2;
```

Comment:
Can we make this more readable ?

Reply:
Oops! Sorry I didn't notice this review message. I will fix it in a separate one.